docs/appendix/apis/components/motor.md

```diff
@@ -6,6 +6,8 @@ type: "docs"
 description: "Give commands to operate a motor or get its current status."
 icon: true
 images: ["/icons/components/motor.svg"]
+date: "2022-10-10"
```

Comment:
did you mean to say 01-01 here and in the ones immediately following? unsure if intentional

Reply:
```suggestion
date: "2022-01-01"
```